api/BUILD

```diff
@@ -29,6 +29,7 @@ proto_library(
         "//envoy/config/filter/fault/v2:pkg",
         "//envoy/config/filter/http/adaptive_concurrency/v2alpha:pkg",
         "//envoy/config/filter/http/buffer/v2:pkg",
+        "//envoy/config/filter/http/cache/v2:pkg",
```

Comment:
Do the config protos need to be in this PR? Can we defer this until you add the filter? Maybe just do the helpers first?

Reply:
Reverted config protos